front/app/containers/Admin/projects/project/nativeSurvey/AccessRightsNotice/index.tsx

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+
+// components
+import { Box } from '@citizenlab/cl2-component-library';
+import Warning from 'components/UI/Warning';
+import Link from 'utils/cl-router/Link';
+
+// hooks
+import usePhasePermissions from 'api/phase_permissions/usePhasePermissions';
+import usePermissionsCustomFields from 'api/permissions_custom_fields/usePermissionsCustomFields';
+import useUserCustomFields from 'api/user_custom_fields/useUserCustomFields';
+import useLocalize from 'hooks/useLocalize';
+
+// intl
+import messages from './messages';
+import { FormattedMessage, useIntl } from 'utils/cl-intl';
+
+type AccessRightsNoticeProps = {
+  projectId: string;
+  phaseId: string;
+};
+
+const AccessRightsNotice = ({
+  projectId,
+  phaseId,
+}: AccessRightsNoticeProps) => {
+  const localize = useLocalize();
+  const { formatMessage } = useIntl();
+  const { data: permissions } = usePhasePermissions({ phaseId });
+  const { data: userCustomFields } = useUserCustomFields();
+  const { data: permissionCustomFields } = usePermissionsCustomFields({
+    projectId,
+    phaseId,
+    initiativeContext: false,
+    action: 'posting_idea',
+  });
+
+  // NOTE: There should only ever one permission for a survey phase
+  const permittedBySetting = permissions?.data[0].attributes.permitted_by;
+  const globalCustomFieldsSetting =
+    permissions?.data[0].attributes.global_custom_fields;
+  const permissionCustomFieldIds = permissionCustomFields?.data.map(
+    (customField) => customField.relationships.custom_field.data.id
+  );
+
+  const surveyUserFields: Array<string | null> | undefined =
+    userCustomFields?.data
+      .filter((customField) => {
+        if (
+          !globalCustomFieldsSetting ||
+          permittedBySetting === 'everyone_confirmed_email'
+        ) {
+          return permissionCustomFieldIds?.includes(customField.id);
+        } else {
+          return customField.attributes.enabled;
+        }
+      })
+      .map((customField) => {
+        return localize(customField.attributes.title_multiloc);
+      });
+
+  const accessRightsSettingsLink = (
+    <Link to={`/admin/projects/${projectId}/phases/${phaseId}/access-rights`}>
+      {formatMessage(messages.accessRightsSettings)}
+    </Link>
+  );
+
+  return (
+    <Box id="e2e-warning-notice" mb="20px">
+      <Warning>
+        {permittedBySetting === 'everyone' ? (
+          <>
+            <p>{formatMessage(messages.anyoneIntro)}</p>
+            <ul>
+              <li>{formatMessage(messages.anyoneBullet1)}</li>
+              <li>{formatMessage(messages.anyoneBullet2)}</li>
+            </ul>
+            <p>
+              <FormattedMessage
+                {...messages.anyoneOutro}
+                values={{
+                  accessRightsSettingsLink: accessRightsSettingsLink,
```

Comment:
Minor. We can use the object short hand here and on line 103
```suggestion
                  accessRightsSettingsLink,
```